### DISCUSSION TITLE
Dropped sliderends count towards combo when using Classic mod in osu! ruleset

### DISCUSSION THREAD

**Discussion by Rian8337** [ppy/osu#15897] [Q&A] [answered]
As the title says. I know the Classic mod is not supposed to completely replicate osu!stable's gameplay mechanics, however this looks like a potential oversight to me.

I tried searching issues with queries `is:issue is:open classic label:area:mods`, `is:issue is:open classic label:area:gameplay`, and `is:issue is:open classic label:ruleset:osu!`, but could not find an open issue for this.

Video replicating the behavior:

https://user-images.githubusercontent.com/52914632/144389906-ca19208b-e6cf-4d2d-90dc-41db20045c44.mp4

[This](https://osu.ppy.sh/scores/osu/3944282350) is the score replayed in the video.

Classic mod customizations remain untouched.

osu!lazer version: 2021.1127.0

Logs:
[performance.log](https://github.com/ppy/osu/files/7640510/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7640511/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7640512/updater.log)
[database.log](https://github.com/ppy/osu/files/7640513/database.log)
[network.log](https://github.com/ppy/osu/files/7640517/network.log)

**Comment by integerrr** [comment]
Afaik classic mod isn't meant to replicate stable behaviour 100% because it's either impossible/infeasible, e.g. it's easier to drop sliderend/get slider 100s in stable than it is in lazer - and this is not about dropping combo

**Reply by Rian8337** [reply]
I know it's not meant to completely replicate stable behavior. Read the opening paragraph of the discussion. I'm merely asking whether this is intended since it looks like an oversight.

**Comment by smoogipoo** [comment, marked as answer]
Yes, this is known. It's because we've simplified scoring in lazer, and there's no longer a type of judgement which does not cause a combo break but causes combo to not be added.

I'm unsure if this will ever return. I'll need to ask for more opinions from higher ranked players, but at the moment it feels better _for me personally_ and I think it's more intuitive for newer players as we regularly have people asking us how they got less combo for a score.

We can keep this discussion open as tracking.

**Reply by Rian8337** [reply]
aha, gotcha. Thanks for the clarification.

**Comment by NotGumballer91** [comment]
I strongly believe this should stay as it is. The classic mod is meant to replicate stable's behavior and it's exactly doing that.

**Reply by peppy** [reply]
See https://github.com/ppy/osu/discussions/15897#discussioncomment-1736089

**Comment by SheepCommander** [comment]
I noticed that even with classic mod on, when I submit a replay to https://ordr.issou.best/ the results screen (Score & Grade) have a tendency to look different compared to what's shown on the Lazer client.

Is this issue the reason that the scores & grade are different?

**Reply by SheepCommander** [reply]
Ex: In Lazer (classic mod on) I get a D on an easy song, but on [issou replay](https://link.issou.best/ITl74) I fail the song ;-;

![image](https://user-images.githubusercontent.com/66646514/220858232-3fea0790-a7bb-4a25-ad48-27460e8f1fcf.png)

**Reply by peppy** [reply]
I have no idea what the linked site is but it's not run by us. Please open an issue on their issue tracker instead.

**Reply by SheepCommander** [reply]
Sorry! You're right, I just thought that third party site might be an example of how Lazer+Classic mod isn't at parity with Stable. Nvm tho, thanks
